### PR TITLE
Add support for JSON.TYPE

### PR DESCRIPTION
**Pull request by gkorland**

### CODE REVIEW

**Comment by gkorland** [comment]
fix #10

**Comment by gkorland** [comment]
@gavrie ?

File: src/redisjson.rs [hunk 1]
```diff
@@ -64,4 +64,30 @@ impl RedisJSON {
 
         return Ok(s)
     }
+
+    pub fn get_type(&self, path: &str) -> Result<String, Error> {
+        let s = match self.get_doc(path)? {
+            Some(doc) => {
+                match *doc {
```
Comment (gavrie) [inline src/redisjson.rs]: This is Rust... no need to dereference explicitly 😄 
Just `match doc {` will do nicely.

File: src/redisjson.rs [hunk 2]
```diff
@@ -64,4 +64,30 @@ impl RedisJSON {
...
+                    Value::Null => "Null",
```
Comment (gavrie) [inline src/redisjson.rs]: Shouldn't this be `"null"` rather than `"Null"`?

**Review by gavrie — CHANGES_REQUESTED** [review]

**Pull request merged**